packages/pds/tests/server.test.ts

```diff
@@ -0,0 +1,44 @@
+import { CloseFn, runTestServer, TestServerInfo } from './_util'
+import axios, { AxiosError } from 'axios'
+
+describe('server', () => {
+  let server: TestServerInfo
+  let close: CloseFn
+
+  beforeAll(async () => {
+    server = await runTestServer({
+      dbPostgresSchema: 'server',
+    })
+    close = server.close
+  })
+
+  afterAll(async () => {
+    await close()
+  })
+
+  it('preserves 404s.', async () => {
+    const promise = axios.get(`${server.url}/unknown`)
+    await expect(promise).rejects.toThrow('failed with status code 404')
+  })
+
+  it('turns unknown errors into 500s.', async () => {
+    // Sneak a new, failing route in before the server's error handler
```

Comment:
not quite sure im following this one 🧐

Reply:
If I add a route to `server.app` without this rigamarole, it will be placed as middlware _after_ the app's error handler.  so any errors from the route wouldn't land in the error handler.  I suppose I could just create a new express app here intead and test the error handler in isolation.  I'll do that!